test/unit_tests/braket/ocean_plugin/conftest.py

```diff
@@ -84,64 +90,64 @@ def advantage_device_parameters():
 def braket_sampler_properties(
     provider_properties, service_properties, two_thousand_q_device_parameters
 ):
-    return DwaveDeviceCapabilities.parse_obj(
-        {
-            "braketSchemaHeader": {
-                "name": "braket.device_schema.dwave.dwave_device_capabilities",
-                "version": "1",
-            },
-            "provider": provider_properties,
-            "service": {
-                "executionWindows": [
-                    {
-                        "executionDay": "Everyday",
-                        "windowStartHour": "11:00",
-                        "windowEndHour": "12:00",
-                    }
-                ],
-                "shotsRange": service_properties["shotsRange"],
-            },
-            "action": {
-                "braket.ir.annealing.problem": {
-                    "actionType": "braket.ir.annealing.problem",
-                    "version": ["1"],
+    d_wave_device_capabilities_json = {
+        "braketSchemaHeader": {
+            "name": "braket.device_schema.dwave.dwave_device_capabilities",
+            "version": "1",
+        },
+        "provider": provider_properties,
+        "service": {
+            "executionWindows": [
+                {
+                    "executionDay": "Everyday",
+                    "windowStartHour": "11:00",
+                    "windowEndHour": "12:00",
                 }
-            },
-            "deviceParameters": two_thousand_q_device_parameters,
-        }
-    )
+            ],
+            "shotsRange": service_properties["shotsRange"],
+        },
+        "action": {
+            "braket.ir.annealing.problem": {
+                "actionType": "braket.ir.annealing.problem",
+                "version": ["1"],
+            }
+        },
+        "deviceParameters": two_thousand_q_device_parameters,
+    }
+    validate(d_wave_device_capabilities_json, DwaveDeviceCapabilities.schema())
+    return DwaveDeviceCapabilities.parse_obj(d_wave_device_capabilities_json)
```

Comment:
We are validating a json blob which we constructed for tests. How do we know if this json represents actual data and not something which has been morphed to get the tests to pass ?